source_frame: "/map"
child_frame: "/base_link"
map_topic: "/map"
method: "tf" # tf or odom only #TODO: combined method
odom_topic: "/odom"
cost_map: true
pointcloud_topic: "/pcl"
global_map_static: false
map_size: [5.0, 3.0] #x,y (meter)
tolerance: 0.3
cost: 0.2 # meter
update_frequency: 10 # Hz